### ISSUE TITLE
Add support for managing AWS Glue Databrew resources

### ISSUE THREAD

**Issue by jussapaavo** [hashicorp/terraform-provider-aws#26324]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

I would like to request a support for managing AWS Glue Databrew resources (datasets, projects, jobs, recipies) with Terraform. This is not currently available through Terraform. However, it is possible through CloudFormation and API. This would be hugely beneficial for us, since we're looking forward to managing all of our stack in Terraform instead of creating some of the resources through web UI.

### New or Affected Resource(s)

* AWS Glue Databrew (NEW)

### References

* [AWS Glue Databrew API documentation](https://docs.aws.amazon.com/databrew/latest/dg/API_Operations.html)
* [AWS Glue Databrew CloudFormation documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/AWS_DataBrew.html)

**Comment by OJFord** [comment]
Is there a particular reason these don't exist, such that I'd run into trying to implement them or that a PR likely wouldn't be accepted?

**Comment by diofeher** [comment]
@OJFord I don't think so, probably is not a priority to anyone at any company, so it just hangs here haha
